extensions/sns/build.rs

```diff
@@ -0,0 +1,16 @@
+const REPLICA_REV: &str = "0062aec2efc16d6e4cadb2cd1052aaabbc9f6e48";
```

Comment:
```suggestion
const REPLICA_REV: &str = "90e2799c255733409d0e61682685afcc2431c928";
```

Just adding something more recent